DOCS.md

```diff
@@ -19,6 +19,11 @@ The following parameters are used to configure this plugin:
 * `secrets` - variables to use in `secret_template`. These are base64 encoded by the plugin.
 * `secrets_base64` - variables to use in `secret_template`. These should already be base64 encoded; the plugin will not do so.
 
+[Istio](https://istio.io/) specific parameters:
+* `kube_inject` *optional*, _{bool}_ - if true, will run `istioctl kube-inject` on the generated kubernetes manifests (defaults to `false`)
```

Comment:
can you name all the options to indicate that they're isto-specific?

Reply:
so, `istio_kube_inject` instead of `kube_inject`?

Reply:
fwiw, they are currently named to align with the `istioctl kube-inject` [subcommand arguments](https://istio.io/docs/reference/commands/istioctl.html#istioctl-kube-inject) ...